Fix / improve non-CRAN dependencies via Remotes (#15)
miraisolutions/techguides: shiny-ci-cd.Rmd

### non-CRAN dependencies

> Deploying a packaged shiny application which uses non-CRAN package sources like Github requires additional information in the `DESCRIPTION` file. Namely, the repository details of such dependencies must be included in a [`Remotes:`](https://cran.r-project.org/web/packages/devtools/vignettes/dependencies.html) field, to allow tools like `renv` or `remotes` to know where the packages should be retrieved from.

Reviewer: ```suggestion
Deploying a packaged shiny application which uses non-CRAN package sources like Github requires additional information in the `DESCRIPTION` file. Namely, the repository details of such dependencies must be included in a [`Remotes:`](https://cran.r-project.org/web/packages/devtools/vignettes/dependencies.html) field,so that tools like `renv` or `remotes` know where the packages should be retrieved from.
```
